xml/System.Security.Cryptography/RSAOpenSsl.xml

```diff
@@ -481,13 +603,18 @@
         <Parameter Name="padding" Type="System.Security.Cryptography.RSASignaturePadding" />
       </Parameters>
       <Docs>
-        <param name="hash">To be added.</param>
-        <param name="signature">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="padding">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hash">The hash value of the signed data.</param>
+        <param name="signature">The signature data to be verified.</param>
+        <param name="hashAlgorithm">The hash algorithm used to create the hash value.</param>
+        <param name="padding">The padding mode.</param>
```

Comment:
This parameter has one extra word, but I think it could also be described better.

Reply:
Same suggestion applies to the rest of the padding params.

Reply:
The "padding" parameter descriptions are the ones that are already present on the documented overloads and variants (e.g. https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsa.signdata?view=netframework-4.8).

They can be long winded, such as "The padding mode to use for the {encryption|decryption|signing|verification} process.", but there's really nothing else that I can think of to say about them.  I'm open to suggestions, but absent a specific suggestion I'm going to leave them as-is.